svthalia/Reaxit
# Add a class for collapsing menu actions

### Summary
This adds a centralized place to implement appbar actions. This gives a central place for common actions like `IconButton`, but also allows others to implement their own.

This resolves some small issues, like inconsistent padding, and tooltips, and also automatically collapses if it becomes more than 3 as specified in MaterialDesign 3, but this can be customized in the future to be page-specific

### How to test
Steps to test the changes you made:
1. Go to go through all the screens to see the buttons still work
